src/pages/index.js

```diff
@@ -68,12 +82,23 @@ class IndexPage extends Component {
           <GameContainer class="mid ">
             <Game
               data={data}
+              points={points}
+              setPoints={this.points}
```

Comment:
potencjalnie zrób inną nazwe na state i na funkcje chociażby setPoints.

Reply:
setPoints to bardzo dobra nazwa

Reply:
zrobione

Reply:
zrobione